### DISCUSSION TITLE
Accessibility Compliance with Scalar API Reference

### DISCUSSION THREAD

**Discussion by kayla-beckham** [scalar/scalar#3420] [General]
Hi,

My team is working on integrating the Scalar API Reference react component to our site, and we had our whole project evaluated for web accessibility. There were a number of issues with the scalar API Reference component - mostly to do with specific aria attributes or the way focus moves around when using keyboard operation on the page. I want to open issues here to share these improvements to accessibility with the scalar team, but I was wondering if you prefer that I put all the items into 1 GitHub issue, or I could split the items by feature or something and make a few 'Issues' posts. Do you have a preference on how I might post the Issues?

Thanks!

**Comment by marclave** [comment]
hey @kayla-beckham appreciate your message!

feel free to make individual issues and we can tackle them as they come! ✨
